src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void execute(Map.Entry<String, Integer> entry, int value) {
+        entry.setValue(value);
+    }
+
+    public static class ReturnOperation implements OperationHandler {
+        @Override
+        public void execute(Map.Entry<String, Integer> entry, int value) {
+            entry.setValue(entry.getValue() + value);
+        }
+    }
+}
```

Comment:
According to the checklist, we should avoid keeping all logic in a single package. The `BalanceOperation` and `ReturnOperation` classes should be placed in separate files within the 'strategy' package to adhere to the Strategy pattern properly.